benchmark-service/benchmark/benchmark.py

```diff
@@ -41,12 +51,21 @@ async def index(request: web.Request) -> Dict[str, Any]:
 
 def init_app() -> web.Application:
     app = web.Application()
+    setup_aiohttp_jinja2(app, 'benchmark')
+    setup_aiohttp_session(app)
+    setup_common_static_routes(router)
+    admin = web.Application()
```

Comment:
So you don't need the admin app anymore. This sub app wrapping is taken care of for you when you call `deploy_config.prefix_application` below.